go.mod

```diff
@@ -9,6 +9,7 @@ require (
 	github.com/davecgh/go-spew v1.1.1
 	github.com/disiqueira/gotree v1.0.0
 	github.com/go-logr/logr v1.2.4
+	github.com/gobwas/glob v0.2.3
```

Comment:
https://pkg.go.dev/path/filepath#Glob can't be used since you want to use `**` right

Reply:
yeah, I've checked stdlib and think that the import makes sense.